src/pages/completed.tsx

```diff
@@ -0,0 +1,66 @@
+import type { FC } from 'react';
+import { useState, useEffect } from 'react';
+import Head from '@/components/head';
+import Layout from '@/components/Layout';
+import Card from '@/components/tasks/card';
+import useFetch from '@/hooks/useFetch';
+import classNames from '@/styles/tasks.module.scss';
+import { task } from '@/components/constants/types';
+import Accordion from '@/components/Accordion';
+
+const TASKS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/tasks`;
+
+const renderCardList = (tasks: task[]) => tasks.map(
+  (item: task) => <Card content={item} key={item.id} />,
+);
+
+const Completed: FC = () => {
+  const [tasks, setTasks] = useState<task[]>([]);
+  const [completeTasks, setCompleteTasks] = useState<any>(null);
+
+  const {
+    response,
+    error,
+    isLoading,
+  } = useFetch(TASKS_URL);
+
+  useEffect(() => {
+    if ('tasks' in response) {
+      setTasks(response.tasks);
+
+      const complete = tasks.filter(
+        (item: task) => item.status === 'Completed',
```

Comment:
`Completed` in lower case?

Reply:
please refer https://github.com/Real-Dev-Squad/website-status/pull/97#discussion_r604332513